.snyk

```diff
@@ -4,9 +4,9 @@ version: v1.25.0
 ignore:
   SNYK-JAVA-IONETTY-5953332:
     - '*':
-        reason: No matching gRPC version is available at this time
-        expires: 2024-01-01T00:00:00.000Z
-        created: 2023-10-20T01:20:35.772Z
+        reason: No gRPC version with a fix is available
+        expires: 2024-03-30T00:00:00.000Z
```

Comment:
I hope there is a solution by this expiry: `2024-03-30T00:00:00.000Z`

Reply:
If not, we can extend it again.